### PR TITLE
DRAFT: crypto: intel: packages are arch specific

### PR DESCRIPTION
**Pull request by erig0**
These packages are x86_64 only.

### CODE REVIEW

File: configs/sst_security_crypto-intel.yaml [hunk 1]
```diff
@@ -5,10 +5,12 @@ data:
...
-  - intel-ipsec-mb
-  - intel-ipp-crypto-mb
-
   labels:
   - eln
   - c9s
+
+  arch_packages:
+    x86_64:
+      - arch-specific-package1
```
Comment (yselkowitz) [inline configs/sst_security_crypto-intel.yaml:14]: This line is a typo.

**Comment by yselkowitz** [comment]
@erig0 also there seems to be some confusion about adding these packages to RHEL: https://src.fedoraproject.org/rpms/qatengine/pull-request/3

File: configs/sst_security_crypto-intel.yaml (hunk 1 above)
Comment (erig0) [reply configs/sst_security_crypto-intel.yaml]: Yikes. copy/paste error. Thanks for catching.

**Comment by erig0** [comment]
> @erig0 also there seems to be some confusion about adding these packages to RHEL: https://src.fedoraproject.org/rpms/qatengine/pull-request/3

Thanks. I'll investigate.

**Comment by yselkowitz** [comment]
The qatengine change which would have used these libraries was rejected.  It seems this can be closed; #954 reverts these additions entirely.

**Pull request closed without merging**